src/wr_logic_ai/nodes/state_machine/state_machine.py

```diff
@@ -1,95 +1,127 @@
+#!/usr/bin/env python
+
 from statemachine import StateMachine, State
```

Comment:
Use the `requirements.txt` to install this package (`python-statemachine==2.0.0`)

Reply:
Addressed in [277b00a](https://github.com/WisconsinRobotics/WRover_Software/pull/48/commits/277b00a47c8537aba6ba2aeeb6c09d9b5ea4d3eb).